language/stdlib/modules/LibraAccount.move

```diff
@@ -150,12 +150,10 @@ module LibraAccount {
         assert(LibraTimestamp::is_genesis(), ENOT_GENESIS);
         // Operational constraint, not a privilege constraint.
         assert(Signer::address_of(lr_account) == CoreAddresses::LIBRA_ROOT_ADDRESS(), EINVALID_SINGLETON_ADDRESS);
-        let limits_cap = AccountLimits::grant_calling_capability(lr_account);
-        AccountLimits::initialize(lr_account, &limits_cap);
         move_to(
             lr_account,
             AccountOperationsCapability {
-                limits_cap,
+                limits_cap: AccountLimits::grant_calling_capability(lr_account),
```

Comment:
maybe this happened for a different reason, but why is it called 'calling capability'? is it used for other caps?

Reply:
Just used for account limits. It's probably a good idea to update the name; I'll make that change in this PR.